pages/components/Modal/modal.html

```diff
@@ -0,0 +1,134 @@
+                <div class="sidebar-heading">Get Started</div>
+                <ul class="sidebar-list">
+                    <li class="sidebar-list-item">
+                        <a href="#">Installation</a>
+                    </li>
+                </ul>
+                <div class="sidebar-heading">Components</div>
+                <ul class="sidebar-list">
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Alerts/alerts.html">Alert</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Avatar/avatar.html">Avatar</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Badge/badge.html">Badge</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Buttons/button.html">Button</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Cards/cards.html">Card</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Image/image.html">Image</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Input/input.html">Input</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="/pages/components/Modal/modal.html">Modal</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="#">Typography</a>
+                    </li>
+                    <li class="sidebar-list-item">
+                        <a href="#">Toast</a>
+                    </li>
+                </ul>
+            </aside>
+            <main class="docs-main-container">
+                <div class="docs-hero-header">
+                    <h1 class="docs-hero-heading">Modal</h1>
+                    <p class="docs-hero-para">
+                        A modal is a window that appears on top of a parent screen.It's called 'modal' because it creates a mode that disables the parent screen but keeps it visible. Users must interact with the modal to return to the main screen.<br/><br/>To use any modal element add the class <b>modal</b>, followed by the variant class names.
+                    </p>
+                </div>
+                <div class="component-variants">
+                    <h3 class="component-varaint-heading">Modal Example</h3>
+                </div>
+                <div class="modal-preview-container">
+                    <div class="modal">
+                        <div class="modal-header">
+                            <h2 class="modal-heading">Modal Title</h2>
+                            <span class="btn-dismiss">X</span>
```

Comment:
It's probably better to use x-icon here. Also give it a color other than white, I needs to be visible even if the user is not hovering over it

Reply:
Made the changes !